model/note/note_test.go

```diff
@@ -27,3 +27,35 @@ foobar **bold**
 	md := markdownSerializer(nil).Serialize(node)
 	assert.Equal(t, initial, md)
 }
+
+func TestText(t *testing.T) {
+	initial := `# My title
+
+hello **world**
+
+- foo
+- bar
+- baz
+`
+
+	expected := `My title
+
+hello world
+
+- foo
+
```

Comment:
Any idea why we end up with these empty lines in lists?

Reply:
Yes, it's because the schema allows to have paragraphs and blocks of code inside a list item